src/mapping.ts

```diff
@@ -337,6 +338,8 @@ export function handleNftFeedUpdate(call: UpdateCall): void {
   let loanIndex = shelf.nftlookup(nftId);
   let loanId = loanIdFromPoolIdAndIndex(poolId, loanIndex)
 
+  // DISCUSS V3: store NAV value here?
```

Comment:
#### Not relevant for MVP

NAV value changes:

1. estimated NAV value on borrow/repay/risk changes ongoing pool (in the subgraph we use this value)
2. precise NAV value on epoch execution